docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,318 @@
+kube-system   kube-controller-manager-minikube   1/1     Running   2 (21m ago)   24m
+kube-system   kube-proxy-hsbj4                   1/1     Running   1 (21m ago)   24m
+kube-system   kube-scheduler-minikube            1/1     Running   2 (21m ago)   24m
+kube-system   storage-provisioner                1/1     Running   3 (19s ago)   24m
+```
+
+For stopping.
+
+```sh
+minikube stop
+```
+```sh
+✋  Stopping node "minikube"  ...
+🛑  Powering off "minikube" via SSH ...
+🛑  1 node stopped.
+```
+
+Or if you want to clear any persistent data please use
+```sh
+minikube delete
+```
+```sh
+minikube stop
+```
+## Deploy `substreams-charts`
+
+### Pull `substreams-charts` release
+
+**NOTE**: If you clone our [substreams-charts Repo](https://github.com/ultraio/substreams-charts) directly, please skip ahead.
+
+You can check out our [substreams-charts Release](https://github.com/ultraio/substreams-charts/releases). Find the release you want to download.
+
+Please note version mentioned next might be out of date by the time you read this, please use the latest version for the most compatible.
+
+Next, you can use this command to pull the charts to your local directory.
+
+```sh
+helm pull https://github.com/ultraio/substreams-charts/releases/download/firehose-antelope-1.0.3/firehose-antelope-1.0.3.tgz --untar
+```
+
+`firehose-antelope-1.0.3.tgz` will be downloaded and unpacked in your current folder under the name `./firehose-antelope`
+
+Additionally, for running a producer node on the same cluster, please pull out another chart called `producer`.
+
+```sh
+helm pull https://github.com/ultraio/substreams-charts/releases/download/producer-0.1.0/producer-0.1.0.tgz --untar
+```
+
+`producer-0.1.0.tgz` will be downloaded and unpacked in your current folder under the name `./producer`
+
+### Deploy `producer` chart
+
+Start your `minikube`
+
+```sh
+minikube start
+```
+
+Once `minikube` is up, you need to deploy `producer` chart, which will contain a `producer` pod that will run `ultratest` out of the box.
+
+```sh
+helm install producer ./producer
+```
+
+Check your deployment
+
+```sh
+kubectl get pod producer-0
+```
+
+Make sure the status is `Running`. You can also use `kubectl get pod producer-0 --watch` to watch the update.
+
+```sh
+NAME         READY   STATUS    RESTARTS   AGE
+producer-0   1/1     Running   0          57s
+```
+
+**NOTE**: If you follow the above instructions, your `producer-0` can be reached at `producer-0.producer.default.svc.cluster.local`. This is important because we will use this IP in `substreams` reader node to connect to the producer to feed block data from it. If for some reason you decide to deploy it in another namespace or under another name, please use this to find its cluster IP `kubectl get pod -o wide`.
+
+### Deploy `firehose-antelope` chart
+
+Before deploying the `firehose-antelope` chart, make sure to create `values.yaml` file to configure your deployment. You can use the file below for your local test:
```

Comment:
I think it would be convenient to have the file included in `firehose-antelope-....tgz`.

Reply:
`firehose-antelope-....tgz` is a chart artifact that should only contain the template and default values.
The overdrive `values.yaml` should always be created by the user since it also depends on which environment they want to deploy.